### ISSUE TITLE
feat(patch): YouTube watch later playlist - auto play off option 

### ISSUE THREAD

**Issue by madgoatz** [revanced/revanced-patches#1599]
### Application

YouTube 

### Issue

Auto playing of the next video in the watch later playlist 

### Patch

The option to not auto play next video in the watch later playlist 

### Motivation

some people have auto play off for general videos... a  option like that would be great for the watch later playlist in the instance that we only want to watch one video and not have it skip to the next

### Acknowledgements

- [X] I have searched the existing issues and this is a new and no duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I filled out all of the requested information in this issue properly.

**Comment by SodaWithoutSparkles** [comment]
Potentially also for the liked videos list. But if so please separate the switches

**Comment by EhabxAlaa** [comment]
Yes, that would be amazing to have, also want it for all playlists

**Comment by StS82** [comment]
This would bei a VERY much appreciated Feature! Have it active on my Desktop and love it!

**Comment by Albond87** [comment]
Bumping this... I would also very much like this option. Is this being worked on?
